apps/homescreen/test/marionette/bookmark_order_test.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+/* global __dirname */
+
+var assert = require('assert');
+var Bookmark = require('../../../../apps/system/test/marionette/lib/bookmark');
+var Server = require('../../../../shared/test/integration/server');
+
+marionette('Homescreen - Bookmark order', function() {
+  var client = marionette.client({
+    profile: require(__dirname + '/client_options_bookmarks.js')
+  });
+  var actions, bookmark, home, server, system;
+
+  suiteSetup(function(done) {
+    Server.create(__dirname + '/fixtures/', function(err, _server) {
+      server = _server;
+      done();
+    });
+  });
+
+  suiteTeardown(function() {
+    server.stop();
+  });
+
+  setup(function() {
+    actions = client.loader.getActions();
+    bookmark = new Bookmark(client, server);
+    home = client.loader.getAppClass('homescreen');
+    system = client.loader.getAppClass('system');
+    system.waitForFullyLoaded();
+    home.waitForLaunch();
+  });
+
+  test('Bookmark order is retained after restart', function() {
```

Comment:
nit: It's not clear to me what's the difference between both tests. They have the same description. Can you add comments to highlight the differences and maybe change the description slightly?